setup.py

```diff
@@ -35,7 +35,7 @@ def get_long_description() -> str:
     author="The Open Microscopy Team",
     author_email="",
     python_requires=">=3",
-    install_requires=["omero-py>=5.6.0", "ome-zarr>=0.3.0"],
+    install_requires=["omero-py>=5.6.0", "ome-zarr>=0.4.0"],
```

Comment:
```suggestion
    install_requires=["omero-py>=5.6.0", "ome-zarr>=0.5.0"],
```

which includes the API addition allowing to preserve the version metadata in the `image-label` attributes